Update README.md (#214)
VirtusLab/besom: README.md

Resources created in `Pulumi.run { ... }` block will be created by Pulumi.

A simple example using Scala CLI:
```scala
//> using scala "3.3.1"
//> using plugin "org.virtuslab::besom-compiler-plugin:0.1.0"
//> using dep "org.virtuslab::besom-core:0.1.0"
//> using dep "org.virtuslab::besom-aws:6.2.1-core.0.1.0"

import besom.*
import besom.api.aws

@main def run = Pulumi.run {
   for
      bucket <- aws.s3.Bucket("my-bucket")
   yield exports(
      bucketUrl = bucket.websiteEndpoint
   )
}
```

> [!NOTE]
> Please pay attention to your dependencies, **only use `org.virtuslab::besom-*`** and not `com.pulumi:*`.
> Besom **does NOT depend on Pulumi Java SDK**, it is a completely separate implementation.

## Tips

> - Pass [`Context`](https://virtuslab.github.io/besom/docs/context) everywhere you are using Besom outside of `Pulumi.run` block with `(using besom.Context)`

Author: ```suggestion
- Whenever you use Besom outside the `Pulumi.run` block, pass [`Context`](https://virtuslab.github.io/besom/docs/context) with `(using besom.Context)`
```
